lib/charms/observability_libs/v0/kubernetes_compute_resources_patch.py

```diff
@@ -158,21 +158,23 @@ def adjust_resource_requirements(
         An adjusted (limits, requests) 2-tuple.
 
     >>> adjust_resource_requirements({}, {})
-    ResourceRequirements(limits={}, requests={})
+    ResourceRequirements(claims=None, limits={}, requests={})
```

Comment:
What's `claims=None`?

Reply:
As I mentioned in the edited PR description, the expected value from `python -m doctest` for each those lines includes `claims=None`.

Reply:
Seems like it's a recent lightkube change.
I was getting the opposite (before `tox --recreate`)
```
Failed example:
    adjust_resource_requirements({"cpu": "1", "memory": "1"}, {"memory": "1G"}, False)
Expected:
    ResourceRequirements(claims=None, limits={'cpu': '1', 'memory': '1'}, requests={'memory': '1', 'cpu': '1'})
Got:
    ResourceRequirements(limits={'cpu': '1', 'memory': '1'}, requests={'memory': '1', 'cpu': '1'})
```

After `tox --recreate` all passed. Thanks!